qiskit/circuit/library/standard_gates/brg.py

```diff
@@ -0,0 +1,68 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""bridge gate."""
+
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.quantumregister import QuantumRegister
+
+
+class BRGGate(Gate):
+    r"""
+    The Bridge gate.
+
+    This implementation is based on Page 2, Figure 2 of [1], with the definition of
+    Bridge_gate which is composed of 4 CNOTs gates.
+
+    [1] Toshinari et al., https://arxiv.org/pdf/1907.02686.pdf
+
+
+    **Circuit symbol:**
+
+    .. parsed-literal::
+
+        q_0: ──■──
+               │
+        q_1:  ───
+             ┌─┴─┐
+        q_2: ┤ X ├
+             └───┘
+
+
+    ..Note::
+
+    The bridge gate is used as a transformation rule in mapping.
```

Comment:
```suggestion
        The bridge gate is used as a transformation rule in mapping.
```
I believe this is what's causing the docs build to fail: https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=30359&view=logs&j=e32fad43-1785-5ab1-d2e9-2c0ab2cb49da&t=10db491d-7a76-589f-3b30-e8d05469d998&l=13812

(You can test this locally with `tox -edocs`.)

Reply:
Okay, Kevin! Thanks for the hint.